dlf/plugins/navigation/locallang.xml

```diff
@@ -33,6 +33,7 @@
 			<label index="zoom-in">Zoom In</label>
 			<label index="zoom-out">Zoom Out</label>
 			<label index="zoom-fullscreen">Fullscreen Mode</label>
+			<label index="addBasket">Add to basket</label>
```

Comment:
Please remove those unused language entries for the navigation plugin. Since there are no code changes to the plugin itself, why would we need those language entries?